komga-webui/src/locales/en.json

```diff
@@ -143,19 +143,19 @@
       "vertical": "Vertical",
       "webtoon": "Webtoon"
     },
-    "tooltip_incognito": "Read progress will not be saved"
```

Comment:
Awkward sentence and I think in error.

Reply:
> Awkward sentence and I think in error.

can you expand on what is awkward, and what do you mean by "i think in error" ?

Reply:
It is functionally equivalent to "the progress that has been read".
I see my alternative has the same problem, but less so?
Could be mitigated by "Your …"

Reply:
still not convinced. I think what you are trying to argue is `read progress` vs `reading progress` ?

Reply:
Old version can mean "the progress that has hitherto been read", and the new version can't. Problem solved, issues avoided.

Reply:
To clarify, this is a tooltip showing over the incognito icon (same as when you browse incognito in any modern browser), and it explains that while reading this book, the read progress will not be saved.

Normally when you read a book, the last page read is saved, so you can read it later from there. But when reading incognito, it doesn't happen.

Hope that gives you enough context.

I am ok with your modification.